### PR TITLE
improve table output

### PR DESCRIPTION
**Pull request by itaysk**
add host pids to container mode, remove unneeded columns from non-verbose, improve width of pid columns

### CODE REVIEW

File: tracee/printer.go [hunk 1]
```diff
@@ -105,28 +107,45 @@ func newEvent(ctx context, argsNames []string, args []interface{}) (Event, error
...
 
 func (p tableEventPrinter) Init() {}
 
 func (p tableEventPrinter) Preamble() {
 	if p.verbose {
-		fmt.Fprintf(p.out, "%-14s %-16s %-12s %-12s %-6s %-16s %-6s %-6s %-6s %-16s %-20s %s", "TIME(s)", "UTS_NAME", "MNT_NS", "PID_NS", "UID", "COMM", "PID", "TID", "PPID", "RET", "EVENT", "ARGS")
+		if p.containerMode {
+			fmt.Fprintf(p.out, "%-14s %-16s %-12s %-12s %-6s %-16s %-11s %-11s %-11s %-16s %-20s %s", "TIME(s)", "UTS_NAME", "MNT_NS", "PID_NS", "UID", "COMM", "PID/host", "TID/host", "PPID/host", "RET", "EVENT", "ARGS")
+		} else {
+			fmt.Fprintf(p.out, "%-14s %-16s %-12s %-12s %-6s %-16s %-5s %-5s %-5s %-16s %-20s %s", "TIME(s)", "UTS_NAME", "MNT_NS", "PID_NS", "UID", "COMM", "PID", "TID", "PPID", "RET", "EVENT", "ARGS")
```
Comment (yanivagman) [inline tracee/printer.go:124]: Using 5 chars for pid is not enough. Just ran Tracee and got pid: 875107 (just an example)

File: tracee/printer.go [hunk 2]
```diff
@@ -105,28 +107,45 @@ func newEvent(ctx context, argsNames []string, args []interface{}) (Event, error
...
+		}
 	} else {
-		fmt.Fprintf(p.out, "%-14s %-16s %-6s %-16s %-6s %-6s %-6s %-16s %-20s %s", "TIME(s)", "UTS_NAME", "UID", "COMM", "PID", "TID", "PPID", "RET", "EVENT", "ARGS")
+		if p.containerMode {
+			fmt.Fprintf(p.out, "%-14s %-6s %-16s %-11s %-16s %-20s %s", "TIME(s)", "UID", "COMM", "PID/host", "RET", "EVENT", "ARGS")
+		} else {
+			fmt.Fprintf(p.out, "%-14s %-6s %-16s %-5s %-16s %-20s %s", "TIME(s)", "UID", "COMM", "PID", "RET", "EVENT", "ARGS")
```
Comment (yanivagman) [inline tracee/printer.go:130]: If removing pids, I suggest keeping tid and not pid, as it's more informative and the process name (comm) already implies that different tids belong to the same process.

File: tracee/printer.go (hunk 1 above)
Comment (itaysk) [reply tracee/printer.go:124]: did you customize your max-pid config? as far as I understand the common max is 32768

File: tracee/printer.go (hunk 2 above)
Comment (itaysk) [reply tracee/printer.go:130]: Don't want to remove pid as it's useful for further analysis. I will keep both tid and pid then

File: tracee/printer.go (hunk 1 above)
Comment (yanivagman) [reply tracee/printer.go:124]: no

Comment (itaysk) [reply tracee/printer.go]: expanded to 7

**Comment by itaysk** [comment]
@yanivagman wdyt about adding separation between args in table?
```
pathname: /etc/ld.so.cache flags: O_RDONLY|O_LARGEFILE dev: 8388610 inode: 533737 
```
we can't use | or : as it's common in our args maybe:
```
<pathname: /etc/ld.so.cache> <flags: O_RDONLY|O_LARGEFILE> <dev: 8388610> <inode: 533737> 
```

**Comment by yanivagman** [comment]
Adding a comma separator is possible,
Maybe with a space before and after the comma so it will be easier to parse?

**Comment by itaysk** [comment]
I think that if someone wan't to parse tracee they should use json output. not sure we should tailor the table out for parsing.

**Comment by itaysk** [comment]
also "comma separated" which is common also for parsing doesn't have a space before the comma

**Comment by yanivagman** [comment]
So let's add a comma then

**Pull request merged**